ipld/write.go

```diff
@@ -14,11 +14,8 @@ import (
 )
 
 // BatchSize defines an amount of IPLD Nodes to be buffered and written at once.
-// This configuration is very database backend specific and the current default(128) is not optimized for the version of
-// Badger we use. We set it to one to avoid test flakiness, as some tests may read for data that was not written yet.
-// TODO(@Wondertan): Find out the perfect value for Badger(e.g. ask PL folks) or migrate to go-car IPLD
-//  storage(preferred).
-const BatchSize = 1
+// TODO(@Wondertan): Should rely on biggest extended block size param instead
+const BatchSize = 16384 // (128*128 so we flush big blocks in one IO write) //
```

Comment:
This const variable can be deleted because we can compute anytime we want to send a batch,  by multiplying `(edsWidth/2) * (edsWidth/2)`.

Reply:
Good idea

Reply:
One thing, I would make it `edsWidth*edsWidth`, as we will store it's full size including what was repaired

Reply:
But also, there will be tree nodes, so the amount of ipld.Node getting written is even higher than edsWidth*edsWidth

Reply:
Not sure though how to calculate the amount of nodes in a binary tree with a formula

Reply:
@liamsi, do you know?

Reply:
For a perfect binary merkle tree the number of all nodes (including the leaves) is always 2^(n+1) - 1 (where n = number of leaves)

Reply:
lgtm